src/test/java/com/config/config.properties

```diff
@@ -0,0 +1,10 @@
+# URLS
+MAIN_PAGE_URL = https://github.com
+
+USER_NAME = ta-user-danchik
+REPOSITORY_NAME = new_repository
+
+SCREENSHOT_FOLDER_PATH = src/test/resources/screenshots
```

Comment:
you don't need it in config. You will not change it and will not use it in CI/CD

Reply:
fixed